bindings/azure/servicebusqueues/servicebusqueues.go

```diff
@@ -130,6 +131,8 @@ func (a *AzureServiceBusQueues) parseMetadata(metadata bindings.Metadata) (*serv
 
 	m.ttl = ttl
 
+	m.QueueName = strings.ToLower(m.QueueName)
```

Comment:
Consider commenting why this case conversion is happening for future maintainers.

Reply:
Good call, added a comment.

Doc issue: https://github.com/dapr/docs/issues/1839

Reply:
The recent force push doesn't look like it contains code changes?